mne/epochs.py

```diff
@@ -1700,7 +1708,7 @@ class EpochsArray(Epochs):
     @verbose
     def __init__(self, data, info, events, tmin=0, event_id=None,
                  reject=None, flat=None, reject_tmin=None,
-                 reject_tmax=None, verbose=None):
+                 reject_tmax=None, baseline=(None, 0), verbose=None):
```

Comment:
this was `None` before and now it's `(None, 0)` by default. Make sure you don't break anything.